modules/deployment/src/test/java/org/mule/runtime/module/deployment/internal/DomainDeploymentTestCase.java

```diff
@@ -1941,6 +1945,74 @@ public void pluginDeclaredInDomainIsAbleToLoadClassesExportedByTheAppWhereItIsUs
     withContextClassLoader(anotherClassLoader, () -> completionCallback.error(new NullPointerException()));
   }
 
+  @Test
+  @Issue("MULE-19376")
+  public void pluginDeclaredInDomainIsAbleToGetResourceWithSameNameInAppAndExtensionFromExtension() throws Exception {
```

Comment:
try to reuse code from the two tests. It creates a plugin, executes a flow and detects that the content retrieved is the same.